src/io/opendss.jl

```diff
@@ -1,11 +1,21 @@
 # OpenDSS parser
 
 
-"parses 'source_id' into Dict containing the 'type', 'name' and active 'phases' of the OpenDSS component"
-function parse_source_id(source_id::String)
-    component_type, component_name, active_phases = split(source_id, '.', limit=3)
-    nodes = [contains(active_phases, "$n") for n in 1:3]
-    return Dict("type"=>component_type, "name"=>component_name, "phases"=>nodes)
+"Structure representing OpenDSS `dss_source_id` giving the type of the component `dss_type`, its name `dss_name`, and the active phases `active_phases`"
+struct dss_source_id
+    dss_type::String
```

Comment:
`AbstractString` may be more flexible.  I recently ran into some type checking bugs with`String` vs `SubString{String}` in Julia v1.0